index.js

```diff
@@ -26,6 +29,21 @@ function getScopedCredentials (nerfed, scope, config) {
     c.alwaysAuth = val === 'false' ? false : !!val
   }
 
+  if (userConfig && userConfig[`${scope}tokenHelper`]) {
+    const helper = userConfig[`${scope}tokenHelper`]
+    if (!path.isAbsolute(helper) || !fs.existsSync(helper)) {
+      throw new Error(`${scope}tokenHelper must be an absolute path, without arguments`)
```

Comment:
Install `@pnpm/error` as a dependency and throw an error with an error code. Something like
```suggestion
      throw new PnpmError('BAD_TOKEN_HELPER_PATH', `${scope}tokenHelper must be an absolute path, without arguments`)
```

Same for the below error.

Reply:
Done